spec/lib/auxiliary/test_helper_spec.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require_relative '../../../lib/auxiliary/time_helper'
+
+RSpec.describe AutoHCK::Helper do
```

Comment:
```suggestion
describe AutoHCK::Helper do
```

For consistency with other files. Not sure what this difference implies though.